aequilibrae/paths/route_choice_set.pyx

```diff
@@ -274,26 +327,35 @@ cdef class RouteChoiceSet:
         cdef:
             RouteSet_t *route_set
             pair[vector[long long] *, vector[long long] *] freq_pair
-            vector[long long] *link_union = <vector[long long] *>nullptr
+            vector[long long] *link_union_scratch = <vector[long long] *>nullptr
+            vector[vector[long long] *] *link_union_set = <vector[vector[long long] *] *>nullptr
             vector[vector[double] *] *cost_set = <vector[vector[double] *] *>nullptr
             vector[vector[double] *] *gamma_set = <vector[vector[double] *] *>nullptr
```

Comment:
Can we rename this so it's clear what gamma stands for? path_overlap or something like that?

Reply:
Done